pkg/storage/iscsiNode.go

```diff
@@ -103,10 +134,30 @@ func (iscsi *iscsiStorage) NodePublishVolume(ctx context.Context, req *csi.NodeP
 	}
 	klog.Infof("attached device at %s", path)
 
-	if connector.Multipath {
-		klog.Info("device is using multipath")
-	} else {
-		klog.Info("device is NOT using multipath")
+	exists := true
+	out, err := exec.Command("ls", "-l", fmt.Sprintf("/dev/disk/by-id/dm-name-3%s", wwn)).CombinedOutput()
```

Comment:
I'm not really concerned about this given the way we deploy in a container (so not recommending you bother to change it now) but it's a good practice to always use full pathnames for external applications called in this way, to reduce the likelihood that an attacker could trick us into invoking malware with the same name but in a different location that appears earlier in $PATH.